src/components/frame/Extensions/themes/index.ts

```diff
@@ -0,0 +1,7 @@
+import { light, dark } from './consts';
+import { IThemes } from './types';
+
+export const themes: IThemes = {
```

Comment:
theme

Reply:
Changed to `themeList`, I think it's more precise